docker/test/fuzzer/Dockerfile

```diff
@@ -38,7 +38,7 @@ COPY * /
 SHELL ["/bin/bash", "-c"]
 CMD set -o pipefail \
     && cd /workspace \
-    && /run-fuzzer.sh 2>&1 | ts "$(printf '%%Y-%%m-%%d %%H:%%M:%%S\t')" | tee main.log
+    && timeout -s 9 1h /run-fuzzer.sh 2>&1 | ts "$(printf '%%Y-%%m-%%d %%H:%%M:%%S\t')" | tee main.log
```

Comment:
This will leave clickhouse-server after KILL, but I guess it is fine.

Reply:
We run it inside container, so it's not a problem, everything will be terminated